Settings-default.toml

```diff
@@ -178,3 +178,7 @@ volumes = { test_data = "test_data/"}
 
 [cache]
 enabled = false
+# storage limit for collecting query results before insertion into the cache
+landing_zone_byte_size = 100_000_000 # 100 MB
```

Comment:
Maybe just define a Cache size and take 10% for the "landing zone"?

Reply:
should this be fixed or configurable?

If it's fixed it may be a problem for very large cache size and only few insertions where much of the cache size will be wasted.

Reply:
Then, I would suggest setting a percentage for this, with 10% being the default.